modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java

```diff
@@ -352,6 +378,30 @@ public WikiPage getDraftPage(long nodeId, String title)
 		return page;
 	}
 
+	/**
+	 * Returns the latest wiki page matching the group and the external
+	 * reference code
+	 *
+	 * @param groupId the primary key of the group
+	 * @param externalReferenceCode the wiki page external reference code
+	 * @return the latest matching wiki page
+	 * @throws PortalException if a portal exception occurred
+	 */
+	@Override
+	public WikiPage getLatestWikiPageByExternalReferenceCode(
```

Comment:
Same here. I think this method should be called getLatestPageByExternalReferenceCode to be consistent with the rest of the methods

Reply:
Done